Week2/week2-solution/solution.txt

```diff
@@ -0,0 +1,116 @@
+1. Create table 
+
+stud(id,name) (use I'd as primary key)
+stud_details(id,city,mobileno) (Use I'd as foreign key)
+-->
+         create table stud
+   	 (
+  	      id int primary key,
+   	      name varchar(20)
+   	 );
+
+
+         create table stud_details
+        (
+  	    id int references stud(id),
+  	    city varchar(20),
+   	    mobileno int
+   	 );
+
+2. Insert minimum 10 records for each table.
+-->
+
+ insert into stud values(1,'om');
+ insert into stud values(2,'sai');
+ insert into stud values(3,'ram');
+ insert into stud values(4,'bhumi');
+ insert into stud values(5,'payal');
+ insert into stud values(6,'jay');
+ insert into stud values(7,'ritu');
+ insert into stud values(8,'shiv');
+ insert into stud values(9,'janvi');
+ insert into stud values(10,'ushma');
+
+ select * from stud;
+OUTPUT:-
+
+1|om
+2|sai
+3|ram
+4|bhumi
+5|payal
+6|jay
+7|ritu
+8|shiv
+9|janvi
+10|ushma
+
+ insert into stud_details values(1,'surat',1234568529);
+ insert into stud_details values(2,'bangalor',7293158529);
+ insert into stud_details values(3,'bardoli',1236975218);
+ insert into stud_details values(4,'vyara',1293754826);
+ insert into stud_details values(5,'bharuch',4596378159);
+ insert into stud_details values(6,'vyara',4935786159);
+ insert into stud_details values(7,'surat',5872693584);
+ insert into stud_details values(8,'surat',9634875921);
+ insert into stud_details values(9,'kamrej',8572693158);
+ insert into stud_details values(10,'kamrej',7592183467);
+
+ select * from stud_details;
+OUTPUT:-

```

Comment:
Can you change the mode of output.? 
If yes how..?

Reply:
yes.we can change the mode of output.as follow:-

.mode table
select * from stud_details;

Reply:
👍

Reply:
we can use . separator command

Reply:
Can you give any example of it

Reply:
we can change separator with .separator command..
for example:-     .separator ","